src/test/suite/stubDemoTutorial.test.ts

```diff
@@ -76,14 +77,16 @@ async function validateSimpleTerminalResponse(terminalName : string, terminalTex
 	if (term) {
 		console.log(`-current terminal = ${term?.name}`);
 		await sendTerminalText(terminalName, terminalText);
-		await delay(delayTime);
-		const term2 = focusOnNamedTerminal(terminalName);
-		await delay(delayTime);
-		let result = await getTerminalOutput(terminalName);
-		console.log(`-validateSimpleTerminalResponse terminal output = ${result}`);
+	
+		const resultValue = await waitUntil(async () => {
+			focusOnNamedTerminal(terminalName);
+			const result = await getTerminalOutput(terminalName);
+			console.log(`-validateSimpleTerminalResponse terminal output = ${result}`);
+			return result.includes(terminalText);
+		}, 5000);
 
 		// we're just making sure we get something back and can see the text we put into the terminal
-		expect(result).to.include(terminalText);
+		expect(resultValue).to.be.true;
```

Comment:
this assertion is no more used. if the `result` doesn't include the `terminalResponse` is not found, there is a timeout exception

Reply:
good point